src/main/proto/org/dependencytrack/policy/v1/policy.proto

```diff
@@ -63,6 +63,8 @@ message Component {
   optional string license_name = 50;
   optional string license_expression = 51;
   optional License resolved_license = 52;
+  // When the component current version last modified.
+  optional google.protobuf.Timestamp current_version_last_modified = 53;
```

Comment:
This name is pretty long and in the code it's called `publishedAt`. Can we rename this to `published_at`?